players/playerInfo.cpp

```diff
@@ -516,3 +519,55 @@ void Player::information(const Player* viewer, bool online) {
     }
 
 }
+
+//*********************************************************************
+//                      cmdChecksaves
+//*********************************************************************
+
+int cmdChecksaves(Player* player, cmd* cmnd) {
+    Player* target = player;
+    bool online = true;
+   
+   std::ostringstream oStr;
+
+    player->clearFlag(P_AFK);
+
+    if(player->isBraindead()) {
+        *player << "You are brain-dead. You can't do that!\n";
+        return(0);
+    }
+
+    if(player->isDm() && cmnd->num > 1) {
+        cmnd->str[1][0] = up(cmnd->str[1][0]);
+        target = gServer->findPlayer(cmnd->str[1]);
+        if(!target) {
+            loadPlayer(cmnd->str[1], &target);
+            online = false;
+        }
+        if(!target) {
+            *player << "That player does not exist.\n";
+            return(0);
+        }
+    }
+
+    oStr << "^c" << (!online ? target->getCName() : "Your") << (!online ? "'s": "") << " current saving throws and amounts gained this level:\n\n";
```

Comment:
You don't need to use an ostringstream, you can stream it directly to the player.

*player << ColorOn << (mostly same shit you did with oStr) << ColorOff;